script/99-imu-3dm-gx4.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEMS=="usb", ATTRS{manufacturer}=="Lord Microstrain",ATTRS{product}=="Lord Inertial Sensor", MODE="0777", GROUP="imu", SYMLINK+="lord_imu"
```

Comment:
Good call. I haven't found a good way to handle multiple gx4's. Also, testing that configuration is difficult.